code/datums/supply_packs.dm

```diff
@@ -355,12 +355,13 @@
 	contains = list(/obj/item/bee_egg_carton = 5)
 	cost = 450
 	containertype = /obj/storage/secure/crate/bee
-	containername = "Honey Production Kit (Cardlocked \[Hydroponics])"
+	containername = "Honey Production Kit"
 	access = access_hydro
 	create(var/sp, var/mob/creator)
 		var/obj/storage/secure/crate/bee/beez=..()
 		for(var/obj/item/bee_egg_carton/carton in beez)
 			carton.ourEgg.blog = "ordered by [key_name(creator)]|"
+		beez.req_access.len = 0
```

Comment:
Would prefer the bee crate was moved from being a child of `/obj/storage/secure` if we're not intending to secure it any more.